cmd/sealer/cmd/cluster/run.go

```diff
@@ -510,3 +517,16 @@ func prepareMaterials(infraDriver infradriver.InfraDriver, imageEngine imageengi
 	}
 	return nil
 }
+
+func checkClusterIsExists(cf clusterfile.Interface) error {
```

Comment:
not good use err to identify the check result. i assume  we could return passed result and error.